packages/grafana-ui/src/components/Table/styles.ts

```diff
@@ -31,21 +32,55 @@ export const getTableStyles = (theme: GrafanaTheme2) => {
         border-right: none;
         padding-right: ${lastChildExtraPadding}px;
       }
-
       &:hover {
+        .cellActions {
+          visibility: visible;
+          opacity: 1;
+        }
+      }
+
+      ${overflowOnHover &&
+      `&:hover {
         overflow: visible;
         width: auto !important;
         box-shadow: 0 0 2px ${theme.colors.primary.main};
         background: ${background ?? rowHoverBg};
         z-index: 1;
-
-        .cell-filter-actions  {
-          display: inline-flex;
+        
         }
-      }
+      }`}
+
       a {
         color: inherit;
       }
```

Comment:
looks like we have two `&:hover` sections.

Reply:
fixed